### PR TITLE
GCP VM Disk Loss Charts

### PR DESCRIPTION
**Pull request by neelanjan00**
Charts and icon for `gcp-vm-disk-loss` experiment.

### CODE REVIEW

File: charts/gcp/gcp.chartserviceversion.yaml [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+  name: gcp 
+  version: 0.1.0
+  annotations:
+    categories: gcp
+spec:
+  displayName: gcp chaos
+  categoryDescription: >
+    gcp contains chaos to disrupt state of gcp resources running part of the gcp services
+  experiments:
+  - gcp-vm-instance-stop
```
Comment (ispeakc0de) [inline charts/gcp/gcp.chartserviceversion.yaml:13]: add an entry for gcp-vm-disk-loss also

File: charts/gcp/gcp.chartserviceversion.yaml [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+  keywords: 
+    - "pods" 
+    - "kubernetes" 
+    - "gcp"
```
Comment (ispeakc0de) [inline charts/gcp/gcp.chartserviceversion.yaml]: add disk keyword

File: charts/gcp/gcp.chartserviceversion.yaml (hunk 1 above)
Comment (neelanjan00) [reply charts/gcp/gcp.chartserviceversion.yaml:13]: Updated

File: charts/gcp/gcp-vm-disk-loss/engine.yaml [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+kind: ChaosEngine
+metadata:
+  name: gcp-disk-chaos
+  namespace: default
+spec:
+  # It can be active/stop
+  engineState: 'active'
+  chaosServiceAccount: gcp-vm-disk-loss-sa
+  # It can be delete/retain
+  jobCleanUpPolicy: 'retain'
```
Comment (uditgaurav) [inline charts/gcp/gcp-vm-disk-loss/engine.yaml]: can we remove jobCleanUpPolicy?

File: charts/gcp/gcp.chartserviceversion.yaml [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+  displayName: gcp chaos
+  categoryDescription: >
+    gcp contains chaos to disrupt state of gcp resources running part of the gcp services
+  experiments:
+  - gcp-vm-instance-stop
+  - gcp-vm-disk-loss
+  keywords: 
+    - "pods" 
+    - "kubernetes" 
+	- "disk"
```
Comment (uditgaurav) [inline charts/gcp/gcp.chartserviceversion.yaml]: some indentation issue

File: charts/gcp/gcp-vm-disk-loss/engine.yaml (hunk 3 above)
Comment (neelanjan00) [reply charts/gcp/gcp-vm-disk-loss/engine.yaml]: Fixed

File: charts/gcp/gcp.chartserviceversion.yaml (hunk 4 above)
Comment (neelanjan00) [reply charts/gcp/gcp.chartserviceversion.yaml]: Fixed

**Review by ispeakc0de — APPROVED** [review]

**Review by uditgaurav — APPROVED** [review]

**Pull request merged**